project/Modules.scala

```diff
@@ -0,0 +1,19 @@
+import sbt._
+import Keys._
+
+object Modules {
+
+  val rootSettings = Seq(
+    name := "BellTower",
+    publishArtifact := false,
+    publishLocal := false
```

Comment:
`false` in `rootSettings` prevents you from publishing only the root, `macros` can be still published